ros2test/ros2test/command/test.py

```diff
@@ -24,9 +28,25 @@ class TestCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         """Add arguments to argparse."""
         launch_testing.launch_test.add_arguments(parser)
+        parser.add_argument(
+            '--disable-isolation', action='store_true', default=False,
+            help='Disable automatic ROS_DOMAIN_ID isolation.'
+            'If ROS_DOMAIN_ID is already set, launch_test will respect and use it.'
+            'If it is not set, a random ROS_DOMAIN_ID that is not being used by another'
```

Comment:
This is subtly misleading.  The ROS_DOMAIN_ID isn't selected randomly, and it will only coordinate with other processes that use the same cooperative mechanism of coordination.  Maybe a better phrasing is
```
If it is not set, a ROS_DOMAIN_ID not being used by another ros2 test will be picked
```
Probably a nitpick, but I'm worried that the help message makes a promise that the implementation can't deliver on

Reply:
Corrected in https://github.com/ros2/ros_testing/pull/2/commits/011c5c2dc628810eb097ca48215782e9f40e1126